articles/virtual-machines/migration-classic-resource-manager-faq.yml

```diff
@@ -140,6 +140,44 @@ sections:
           I got the message "VM is reporting the overall agent status as Not Ready. Hence, the VM cannot be migrated. Ensure that the VM Agent is reporting overall agent status as Ready" or "VM contains Extension whose Status is not being reported from the VM. Hence, this VM cannot be migrated."
         answer: |
           This message is received when the VM does not have outbound connectivity to the internet. The VM agent uses outbound connectivity to reach the Azure storage account for updating the agent status every five minutes.
+      
+      - question: |
+          How do I migrate my classic storage accounts to ARM?
+        asnswer: |
+          Migration can be done either using the Azure Portal, with PowerShell or with Azure CLI.
+          1. [Migrate using PowerShell](https://docs.microsoft.com/azure/virtual-machines/migration-classic-resource-manager-ps#step-52-migrate-a-storage-account)
+          2. [Migrate using Azure CLI](https://docs.microsoft.com/azure/virtual-machines/migration-classic-resource-manager-cli#step-5-migrate-a-storage-account)
+          3. [Migrate using Azure Portal](https://docs.microsoft.com/azure/virtual-machines/migration-classic-resource-manager-overview#migration-of-storage-accounts)
+        
+      - question: |
+          What is the official date classic storage accounts will be cut-off from creation?
+        answer: |
+          Subscriptions created after August 2022 will no longer be able to create classic storage accounts. However, any existing subscription before August 2022 can continue to create and manage classic storage resources until the retirement date of 31 August 2024.
+
+      - question: |
+          What happens to existing classic storage accounts after 31 August 2024?
+        answer: |
+          Once the deadline arrives you will no longer be able to access your classic storage accounts and will be forced to migrate them to ARM.
```

Comment:
If customers cannot access the storage account how will they migrate it? Needs more clarification on what exactly customers will and will not be able to do.